Entity Explorer - Host.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>Maybe a comment on what the "user_name" parameter is going to be used for if supplied?</p><p>Not sure if we need a widget to confirm the choice of user name for interactive users (in the section that looks at logons)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>